### ISSUE TITLE
Can’t get menu item 1 of menu "Workspace"

### ISSUE THREAD

**Issue by spazbg** [samknight/slack_applescript#41]
After the latest Slack UI updates, the command "focus workspace 'Your Workspace'" returns this error:

error "System Events got an error: Can’t get menu item 1 of menu \"Workspace\" of menu item \"Workspace\" of menu \"File\" of menu bar item \"File\" of menu bar 1 of process \"Slack\" whose name starts with \"Your Workspace\". Invalid index." number -1719

**Comment by samknight** [comment]
I've not been able to replicate this error - can I confirm that you see the menu bar

File -> Workspace -> [name of workspace]

**Comment by spazbg** [comment]
Yes, my mistake. It seems that the name of the workspace was changed without my knowledge
